newuser.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<meta charset="UTF-8">
+<title>newuser</title>
+<meta name="viewport" content="width=device-width,initial-scale=1">
+<link rel="stylesheet" href="">
+<style>
+</style>
+<script src=""></script>
+<body>
+    <form>
+        <label for="fname">First name:</label><br>
+        <input type="text" id="fname" name="fname" minlength="1" maxlength="32" required><br>
+        <label for="lname">Last name:</label><br>
+        <input type="text" id="lname" name="lname" minlength="1" maxlength="32" required><br>
+        <label for="email">Email:</label><br>
+        <input type="text" id="email" name="email" required><br>
+        <label for="pass">Password:</label><br>
+        <input type="text" id="pass" name="pass" required><br>
```

Comment:
For the more specialized fields, like password and phone, it is usually better to use the specific input type so that the browser can do the work.  For example, setting type=“password” result in the browser showing a password text field where all the characters are ‘*’ to prevent shoulder surfing.